File: WebSolution/PMSNGApp/src/app/time-sheet/models/TimeSheet.ts
import { TimeSheetEntry } from "./TimeSheetEntry";

export class TimeSheet{
    constructor(
        public id:number,
        public status:string,
        public timeSheetDate:string,
        public statusChangedDate:string,
        public employeeId:number
        ){
        }
        public timeSheetEntries:TimeSheetEntry[]=[]
}

File: WebSolution/Monolithic/Intranet/Models/TimeSheet.cs
namespace TFLPortal.Models;

public class Timesheet
{
    public int Id { get; set; }
    public string? Status { get; set; }
    public DateTime TimesheetDate { get; set; }
    public DateTime? ModifiedOn { get; set; }
    public int EmployeeId { get; set; } //employeeId
    public double? TotalHours { get; set; }
}
